sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -401,30 +401,34 @@
   position: inherit;
 }
 
-.img-welcome-background {
-  background-image: url("../images/image_institution_welcome.svg");
+.img-background {
   background-repeat: no-repeat;
   background-size: contain;
   width: 100%;
   height: 100%;
+}
+
+.img-welcome-background {
+  @extend .img-background;
+  background-image: url("../images/image_institution_welcome.svg");
   min-height: $image-min-height;
```

Comment:
is min-height not a common property? is it specific for each of image class?

Reply:
not for all classes